demos/1_Introduction/04_FBP_CGLS_SIRT.ipynb

```diff
@@ -368,7 +367,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Alternatively, tolerance can be used as a stopping criterion."
+    "If we run a lot of iterations we can check if the algorithm converges. We set the `tolerance` to 0 to demonstrate the algorithm continuing to run (but this way of setting tolerance is being deprecated)."
```

Comment:
What will we use instead in the future in terms of the tolerance? Sorry I've forgotten

Reply:
I think we can use callbacks to allow early stopping if tolerance is reached

Reply:
It will be deprecated in 24.2 I think... and will be replaced by a callback